android/build.gradle

```diff
@@ -1,12 +1,12 @@
 buildscript {
-    ext.kotlin_version = '1.6.10'
+    ext.kotlin_version = '1.8.21'
```

Comment:
did you confirm this changing of kotlin version with the Android leads (Priyanshu or Pranjal)

Reply:
Yes